src/main/java/picard/sam/FilterSamReads.java

```diff
@@ -276,6 +298,9 @@ protected int doWork() {
             final SamReader samReader = SamReaderFactory.makeDefault().referenceSequence(REFERENCE_SEQUENCE).open(INPUT);
             final FilteringSamIterator filteringIterator;
 
+            // Used for exclude/include tag filter
+            List<Object> tagList = TAG_VALUE == null ? null: TAG_VALUE.stream().map(T -> (Object) T).collect(Collectors.toList());
```

Comment:
I had to add a null check here for the tests testing the other filters to pass, this will never be null when run as a clp.  I just feel slightly iffy about what I've done.